backend/fms_core/migrations/0039_v3_9_0.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 3.1 on 2021-10-21 21:20
+
+from django.conf import settings
+from django.db import migrations, models
+from django.contrib.auth.models import User
+import reversion
+
+ADMIN_USERNAME = 'biobankadmin'
+
+
+def add_property_types_for_library_qc(apps, schema_editor):
+    Protocol = apps.get_model("fms_core", "Protocol")
+    PropertyType = apps.get_model("fms_core", "PropertyType")
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        admin_user_id = admin_user.id
+
+        reversion.set_comment("Add property type 'Library Size' for library QC")
+        reversion.set_user(admin_user)
+
+        # Add a "Library Size" property type for library QC process measurements,
+        # associated with the Sample Quality Control protocol
+
+        protocol_content_type = ContentType.objects.get_for_model(Protocol)
+        sample_qc_protocol = Protocol.objects.get(name='Sample Quality Control')
+       
+        property_type = PropertyType.objects.create(name="Library Size",
```

Comment:
You have to create all the properties not only the ones that are different from sample QC even if they share the same name.

Reply:
Ah, okay. I misunderstood. I thought we wanted to use the same protocol and properties for sample QC and library QC.